motioneye/utils/__init__.py

```diff
@@ -605,5 +604,5 @@ def call_subprocess(args, stdin=None, input=None, stdout=subprocess.PIPE, stderr
     return subprocess.run(
         args, stdin=stdin, input=input, stdout=stdout, stderr=stderr, capture_output=capture_output, shell=shell,
         cwd=cwd, timeout=timeout, check=check, encoding=encoding, errors=errors, text=text, env=env,
-        universal_newlines=universal_newlines
+        text=universal_newlines
```

Comment:
```suggestion
        universal_newlines=universal_newlines
```

Reply:
@cclauss 
Looks like pyupgrade is unhappy with this reverted but it is an obvious syntax error 😄. Looks like a bug in pyupgrade?